packages/__docs__/buildScripts/utils/getClientProps.mts

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 
-import type { LibraryOptions, ParsedDoc, ProcessedFile } from '../DataTypes'
+import type { LibraryOptions, ParsedDoc, ProcessedFile } from '../DataTypes.mjs'
```

Comment:
This rewrite was needed because I thought that an issue would be fixed by upgrading react-docgen... it was not, and it required that our TS files are emitted with ESM imports. This needed a change in the extension otherwise they'd be treated as commonJS files.